docs/README.md

```diff
@@ -778,22 +778,22 @@ The preferred way to create a command is by using the generator, e.g.
 boost new:command CreateProduct --fields sku:SKU displayName:string description:string price:Money
 ```
 
-The generator will automatically create a file called `CreateProduct.ts` with a TypeScript class of the same name under the commands directory. You can still create the command manually. Since the generator is not doing any *magic*, all you need is a class decorated as `@Command`. Anyway, we recommend you always to use the generator, because it handles the boilerplate code for you.
+The above generator will automatically create a file called `CreateProduct.ts` with a TypeScript class of the same name under the `commands` directory. You can still create (or modify) the command manually. Since the generator is not doing any *magic*, all you need is a class decorated as `@Command`. Anyway, we recommend you always to use the generator, because it handles the boilerplate code for you.
```

Comment:
@alvaroloes, something I've learned is to avoid explicit mentions to the position of the tables, snippets, graphics, etc. because if you move them, you don't have to change the texts with the reference.

```suggestion
The generator will automatically create a file called `CreateProduct.ts` with a TypeScript class of the same name under the `commands` directory. You can still create (or modify) the command manually. Since the generator is not doing any *magic*, all you need is a class decorated as `@Command`. Anyway, we recommend you always to use the generator, because it handles the boilerplate code for you.
```

I learned it using LaTex because it is in charge of placing the visual elements wherever they fit better, so you actually don't know where they'll be after the build. I know we are not using LaTex here but I still consider it a very good tip as we could be placing snippets in a different position after building the final docs

Reply:
Amazing tip Johan. 100% agree